src/main/kotlin/com/cognifide/gradle/aem/api/AemConfig.kt

```diff
@@ -456,7 +456,7 @@ class AemConfig(
     }
 
     fun remoteInstance(httpUrl: String) {
-        instance(RemoteInstance.create(httpUrl))
+        remoteInstance(httpUrl, {})
```

Comment:
if env was not explicitly defined, effectively bad default value was used / `cmd`